spec/resque/integration/unique_spec.rb

```diff
@@ -88,7 +88,7 @@ class JobDequeueTest
 
     @queue = :queue2
 
-    def self.execute(x)
+    def self.execute(*x)
```

Comment:
Naming/UncommunicativeMethodParamName: Method parameter must be at least 3 characters long.